apps/aechannel/src/aesc_fsm.erl

```diff
@@ -845,7 +851,7 @@ dep_signed(cast, {?DEP_LOCKED, Msg}, #data{latest = {deposit, SignedTx,
             close(Error, D)
     end;
 dep_signed(timeout, dep_signed = T, D) ->
```

Comment:
This function clause could be removed entirely because it is handled further down by `handle_common_event` too.